index.html

```diff
@@ -1,34 +1,62 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
 
 <head>
-<title>PDF2SVG</title>
-<meta charset="UTF-8">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="description" content="Convert PDF to SVG , convert SVG to PNG format.">
+  <link rel="stylesheet" href="https://white-ui.whitecoode.com/w29h4i1t87e4-u1icss/s0he98et(css)/whiteuicss.min.css">
```

Comment:
Hi, loading code from CDNs is not okay. C.f. https://github.com/toollabs/validator/pull/1/commits/998e06f841f23b8989f3e3a7490678f4e0adac6f

Reply:
okay